Buildings/Fluid/HeatExchangers/Examples/WetCoilCounterFlowPControlAutoTuning.mo

```diff
@@ -104,11 +109,21 @@ model WetCoilCounterFlowPControlAutoTuning
     "Reset signal"
     annotation (Placement(transformation(extent={{-80,160},{-60,180}})));
   Buildings.Controls.OBC.CDL.Logical.Sources.Pulse autTunSig(
```

Comment:
In a real control scenario, is there any guideline of when should the controller be tuned? When we are saying auto-tuning, it gives the impression that the controller will automatically tuning the index when certain condition is (or is not) satisfied.

This example triggered the tuning process just based on time, which it may start earlier, or later in reality.

Reply:
This is a very good question. The reference says "the system is at equilibrium at the working point (u0, y0) before
the relay experiment is started" but I am not sure how to define the equilibrium. 
Can you please let me know if you have any suggestion?

For the studied example, if we make the tuning starts at the very beginning, the result don't change much.